llvm-external-projects/iree-dialects/lib/Dialect/LinalgExt/IR/LinalgExtOps.cpp

```diff
@@ -1770,6 +1777,31 @@ inferPackedType(ShapedType sourceType, ArrayRef<int64_t> innerTiles,
 // PackOp
 //===----------------------------------------------------------------------===//
 
+/// Custom builder methods for pack ops.
+void PackOp::build(OpBuilder &builder, OperationState &state, Value source,
+                   Value output, ArrayRef<int64_t> innerDimsPos,
+                   ArrayRef<OpFoldResult> innerTiles,
+                   Optional<Value> paddingValue,
+                   ArrayRef<int64_t> outerDimsPerm) {
+  assert(innerDimsPos.size() == innerTiles.size());
+  DenseMap<int64_t, OpFoldResult> tileAndPosMapping;
```

Comment:
We can use `getDimAndTileMapping`. It is already available in the op.

Reply:
I was actually trying to see if we can avoid using `getDimAndTileMapping` to avoid creating a `DenseMap`.